external-cloud-api/src/com/stnetix/ariaddna/externalcloudapi/cloudinterface/iAbstractCloud.java

```diff
@@ -0,0 +1,66 @@
+package com.stnetix.ariaddna.externalcloudapi.cloudinterface;
+
+/**
+ * This is an abstract interface containing general methods for manipulation with any external cloud storage.
+ * Every certain cloud storage realization must implement this interface.
+ */
+public interface iAbstractCloud {
```

Comment:
I think we need next methods:
String moveFile(String from, String to)
String copyFile(String from, String to)